src/main.c

```diff
@@ -671,7 +683,8 @@ static bool check_faults(data *d) {
 static void calculate_setpoint_target(data *d) {
     float input_voltage = VESC_IF->mc_get_input_voltage_filtered();
 
-    if (input_voltage < d->float_conf.tiltback_hv) {
+    if ((input_voltage < d->float_conf.tiltback_hv) &&
```

Comment:
I know lisp people like brackets, but you don't need to add them here :smile:. `<` has higher precedence than `&&`. You can always check here: https://en.cppreference.com/w/c/language/operator_precedence

It's good to add brackets when it's not obvious, but this is one of the most common cases and everyone should know what's the order. The brackets add noise.

Reply:
ah i think dado added that check. i'll change the brackets